drivers/unix/file_access_unix.cpp

```diff
@@ -148,30 +148,6 @@ Error FileAccessUnix::_open(const String &p_path, int p_mode_flags) {
 	return OK;
 }
 
-void FileAccessUnix::close() {
```

Comment:
Maybe keep it as private `_close` for consistency with other implementations, even if used only once.

Reply:
Or not sure actually, I see it's not 100% consistent in other classes either.

IMO it could be worth making sure that all destructors are all just calling `_close();`, but alternatively the few classes which do need to call `_close()` outside the destructor could maybe call the destructor?

Otherwise it can also just be kept as in the current state of this PR, it's also fine :)

Reply:
It seems like in case of Android, not calling `_close` from `_open` is a mistake, if `_open` is called on existing `FileAccess` file won't be properly closed.

`FileAccessUnix` are closing file in the `_open`, but `close` also have "backup save" login in it, co it's probably should also call `_close`.

Reply:
And, I'm not sure if calling `_open` on already opened `FileAccess` is used at all, or should be supported.